tests/ethereum/test_regressions.py

```diff
@@ -353,6 +353,40 @@ def test_mulmod(self):
         # 0x8000000000000000000000000000000000000000000000000000000082000011
         self.assertEqual(Z3Solver.instance().get_all_values(constraints, result), [2423129])
 
+    def test_related_to(self):
+        import gzip
+        from manticore import config
+        from manticore.core.smtlib.visitors import translate_to_smtlib
+        from manticore.core.smtlib import ConstraintSet, Z3Solver, Operators, BitVecConstant
+        import pickle, sys
+        filename = os.path.abspath(os.path.join(DIRPATH, "data", "ErrRelated.pkl.gz"))
+
+        constraints, constraint = pickle.loads(gzip.open(filename,'rb').read())
+
+        consts = config.get_group("smt")
```

Comment:
(There is a contextmanager in config.py to test with different config vars and then restore them to original values)